server/api/helpers/auth/verify-jwt.js

```diff
@@ -37,10 +37,16 @@ module.exports = {
         ...result
       } = user;
 
+      const whitelistData = await Whitelist.findOne({
+        eventName: 'ignite-2022',
+        data: { contains: valid.emailAddress },
+      });
+      result.whitelisted = whitelistData ? true : false;
```

Comment:
since this is not going into release branch, i think it's fine to just leave it, but this should be a bit more descriptive, e.g. igniteWhitelisted